www/source/tutorials/sample-app/mac/build-package.html.slim

```diff
@@ -49,6 +49,8 @@ section.sample-app
 
       p The package created is also stored in a relative <code>results</code> directory (for example, <code>/src/results/originname-myrailsapp-0.1.0-20170624214244-x86_64-linux.hart</code>) that persists when you exit the studio. You will mostly refer to the .hart file when uploading your package to the depot.
 
+      p <b>Note</b> If you get build errors involving nokogiri, follow their #{link_to 'troubleshooting steps', 'http://www.nokogiri.org/tutorials/installing_nokogiri.html#mac_os_x'} before re-running <code>build</code> again. 
```

Comment:
Shouldn't we use `blockquote Note:...` here instead to be consistent with what we have elsewhere for things like this? 

Also Nokogiri should probably be capped here.